gapic-generator-java/DEVELOPMENT.md

```diff
@@ -155,6 +155,32 @@ To generate a production GAPIC API:
    ```
    and then rebuild gapic-generator-java (`mvn clean install`).
 
+## Debugging the Plugin under googleapis with local gapic-generator-java (end-to-end debugging)
+
+To debug the local generator starting from protobuf compiler plugin
+
+1. Add [Remote JVM debug configuration](https://www.jetbrains.com/help/idea/tutorial-remote-debug.html) and breakpoints for the local generator in Intellij.
+
+2. In [googleapis](https://github.com/googleapis/googleapis) root directory, run the following commands:
+   1. Set `JVM_DEBUG_PORT` environment variable.
+       ```shell
+       export JVM_DEBUG_PORT=5005
+       ```
```

Comment:
Can you add why it works?

The protoc runs the "protoc-gen-java_gapic" shell script (a wrapper of java command with an option to specify JVM debugging). Because this script does not initialize `JVM_DEBUG_PORT` variable, we can pass this environment variable to the script.

Reply:
done